packages/mask/src/plugins/Savings/SNSAdaptor/SavingsTable.tsx

```diff
@@ -0,0 +1,167 @@
+    useAsync(async () => {
+        for (const protocol of SavingsProtocols) {
+            await protocol.getApr()
+            await protocol.getBalance(chainId, web3, account)
+        }
+    }, [chainId])
+
+    return (
+        <Box className={classes.containerWrap}>
+            <Grid container spacing={0} className={classes.tableHeader}>
+                <Grid item xs={4} className={classes.tableCell}>
+                    <Typography variant="body1">{t('plugin_savings_type')}</Typography>
+                </Grid>
+                <Grid item xs={2} className={classes.tableCell}>
+                    <Typography variant="body1"> {t('plugin_savings_apy')}</Typography>
+                </Grid>
+                <Grid item xs={3} className={classes.tableCell}>
+                    <Typography variant="body1">{t('plugin_savings_wallet')}</Typography>
+                </Grid>
+                <Grid item xs={3} className={classes.tableCell}>
+                    <Typography variant="body1">{t('plugin_savings_operation')}</Typography>
+                </Grid>
+            </Grid>
+
+            {mappableProtocols.map((categorizedProtocol) => {
+                const protocols = categorizedProtocol.protocols
+                if (protocols.length === 1) {
+                    const protocol = protocols[0]
+
+                    return (
+                        <Grid container spacing={0} className={classes.tableRow} key={protocol.type}>
+                            <Grid item xs={4} className={classes.tableCell}>
+                                <div className={classes.logoWrap}>
+                                    <img src={IconURLs[protocol.category]} className={classes.logo} />
+                                    <img src={IconURLs[protocol.image]} className={classes.logoMini} />
+                                </div>
+                                <div>
+                                    <Typography variant="body1">{protocol.category.toUpperCase()}</Typography>
+                                    <Typography variant="body1" className={classes.protocolLabel}>
+                                        {protocol.name}
+                                    </Typography>
+                                </div>
+                            </Grid>
+                            <Grid item xs={2} className={classes.tableCell}>
+                                <Typography variant="body1">{protocol.apr}%</Typography>
+                            </Grid>
+                            <Grid item xs={3} className={classes.tableCell}>
+                                <Typography variant="body1">
+                                    <FormattedBalance
+                                        value={protocol.balance}
+                                        decimals={protocol.decimals}
+                                        significant={6}
+                                        minimumBalance={leftShift(10, protocol.decimals - 6)}
```

Comment:
You meant to use `rightShift`, right?
```suggestion
                                        minimumBalance={rightShift(10, protocol.decimals - 6)}
```